sunrise.el

```diff
@@ -1793,11 +1801,14 @@ Emacs window configuration into a default state."
   (interactive)
   (and (sunrise-running-p)
        (progn (setq sunrise-current-frame nil)
+              (setq sunrise-last-window-configuration (current-window-configuration))
               (sunrise-save-directories)
               (sunrise-save-panes-width)
-              (when (or norestore (not (sunrise-restore-prior-configuration)))
-                (sunrise-select-viewer-window)
-                (delete-other-windows))
+              (if (or norestore (not (sunrise-restore-prior-configuration)))
+                  (progn
+                    (sunrise-select-viewer-window)
+                    (delete-other-windows))
+                (set-window-configuration sunrise-prior-window-configuration))
```

Comment:
This should be redundant, but I was noticing different behavior when I explicitly put this instead of just relying on `(sunrise-restore-prior-configuration)` from above. Will continue to test....